manifests/init.pp

```diff
@@ -102,18 +102,19 @@
     ssl_cert_db            => $::certs::nss_db_dir,
     ssl_cert_password_file => $::certs::qpid::nss_db_password_file,
     ssl_cert_name          => 'broker',
+    interface              => 'lo',
```

Comment:
That's puppet-qpid, this is puppet-katello. I don't see it being passed on to that module.